pkgs/racket-doc/file/scribblings/glob.scrbl

```diff
@@ -117,6 +119,19 @@ Examples:
   This operation accesses the filesystem.
 }
 
+@defproc*[([(glob-quote [pattern string?]) string?]
+           [(glob-quote [pattern path?]) path?]
```

Comment:
Why are these called `pattern`? They won't be interpreted as patterns, but literal "quoted" strings or paths, right?